### PR TITLE
update auth types

### PR DESCRIPTION
**Pull request by israx**

### CODE REVIEW

File: RFC Draft.md [hunk 1]
```diff
@@ -167,37 +167,25 @@ Amplify is proposing the following changes for the `Auth` category. Similar chan
 
 User attributes inference on the `signUp` API.
 
-**Current Usage (v5)**
-
```
Comment (jimblanc) [inline RFC Draft.md]: Don't we still want this header?

File: RFC Draft.md [hunk 2]
```diff
@@ -167,37 +167,25 @@ Amplify is proposing the following changes for the `Auth` category. Similar chan
...
+    },
+  },
 });
 ```
 
 **Current DX (v5)**
 
 ![signup-v5](https://user-images.githubusercontent.com/70438514/222456624-b4af7349-db9d-4304-9cdc-08cf649e2d31.png)
 
-**Proposed Usage (v6)**
```
Comment (jimblanc) [inline RFC Draft.md]: Ditto here for the proposed v6 usage? Or is the idea v5 & v6 are the same now?

File: RFC Draft.md (hunk 1 above)
Comment (israx) [reply RFC Draft.md]: included

File: RFC Draft.md (hunk 2 above)
Comment (israx) [reply RFC Draft.md]: included usage for v5 and v6

**Review by jimblanc — APPROVED** [review]

**Pull request merged**